contrib/k8s/examples/cosmosdb/cosmosdb-instance.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
   name: my-cosmosdb-instance
```

Comment:
Actually... _after_ thinking the file name should change, maybe should this resource name also?